### ISSUE TITLE
Error creating locales on OS X

### ISSUE THREAD

**Issue by benstadin** [valhalla/odin#369]
I'm beginning to port thor to iOS. When converting the locales as below, I get the following error.

$ cd locales && ./make_locales.sh *.json > ../src/odin/locales.h
Can't open UTF-8: No such file or directory

The locales.h file seems to be created, though I can't verify if anything else is broken.

**Comment by kevinkreiser** [comment]
correct. the software will work but some of the tests will fail. this is because the tests rely on locales and since it seems odd for us to say hey install these locales we use a debian (and possibly other distros) method to make locales files on the fly. if you know how to do that in macos feel free to PR. otherwise you can safely ignore the test failures.
